package.json

```diff
@@ -125,10 +125,19 @@
     "yargs": "6.6.0"
   },
   "dependencies": {
+    "@angular/common": "4.0.0-rc.2",
```

Comment:
Hmm... I think it works the same in either case since we are always using `yarn install` in each environment. I installed them as `dependencies` just to follow the conventions that Angular follows in all of their `package.json`s.

From the [cli](https://github.com/angular/angular-cli/blob/master/package.json), for example, looks like they partition the actual dependencies that compose the application in `dependencies` and those that exist outside of a bundle in `devDependencies`.

I can move everything to `devDependencies` though for consistency.